cmd/lncli/commands.go

```diff
@@ -118,6 +129,7 @@ var newAddressCommand = cli.Command{
 }
 
 func newAddress(ctx *cli.Context) error {
+	ctxb := getContext()
```

Comment:
The commands have been split up into multiple files since this PR was originally created. Would be great to also apply these changes to the commands in the other files (just one example: `cmd/lncli/cmd_pay.go`).